core/templates/dev/head/app.js

```diff
@@ -1220,3 +1220,20 @@ oppia.factory('codeNormalizationService', [function() {
     }
   };
 }]);
+
+// Service for retrieving algorithm id of classifier for a given interaction id.
+oppia.factory('algorithmIdRetrievalService', [function() {
+  // Maintain the Interaction-Classifier mapping as defined in feconf.
+  var interactionClassifierMapping = {
```

Comment:
@prasanna08 -- Yes, I think that's fine. The algorithm ID is already stored with the job results model that contains the classifier data, right? So no additional retrievals are needed?

@pranavsid98 -- I think that makes sense, but it'd be good to get @prasanna08's feedback.